sklearn/model_selection/_search.py

```diff
@@ -882,7 +886,7 @@ class GridSearchCV(BaseSearchCV):
         FitFailedWarning is raised. This parameter does not affect the refit
         step, which will always raise the error.
 
-    return_train_score : boolean, default=True
+    return_train_score : boolean, default=None
```

Comment:
Drop the default here, say optional, and in the description note that the current default, true, will be changed to false in...